mst/aiven-client/grafana-authentication-plugins

```diff
@@ -0,0 +1,201 @@
+---
+title: Grafana authentication plugins
+excerpt: Configure Google, Github, or GitLab authentication integration to Grafana service on fully managed services on AWS, Azure, or GCP.
+product: mst
+---
+
+# Grafana authentication plugins
+
+Grafana supports multiple different authentication plugins, in addition to
+built-in username and password authentication.
+
+Grafana service on MST has support for Google, Github and GitLab authentication.
+You can configure authentication integration using the Aiven command-line client.
+
+## Integrate Google authentication plugin
+
+To integrate Google authentication plugin with Grafana service on Managed Service
+for TimecaleDB, you need to first, create your [Google OAuth
+keys][google-oauth-keys] and make a note of your client ID and client secret.
+
+<procedure>
+
+### Integrating Google authentication plugin
+
+1.  In the Aiven Client, log in to your
+    [Managed Service for TimescaleDB dashboard][mst-login].
```

Comment:
Using this link seems confusing, because you're asking them to log in at the command line, not in the URL you're providing. Maybe a link to the Aiven client page works better here?